test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class SortCommandTest {
+
+    @Test
+    public void execute() throws Exception {
+
+        TypicalPersons td = new TypicalPersons();
+        AddressBook addressBook = TestUtil.createAddressBook(td.dan, td.candy, td.bill, td.amy);
+        List<ReadOnlyPerson> original = addressBook.getAllPersons().immutableListView();
+
+        SortCommand command = new SortCommand();
+        command.setData(addressBook, original);
+        CommandResult result = command.execute();
+
+        assertTrue(result.getRelevantPersons().isPresent());
```

Comment:
There may be no relevant persons (e.g. if you tried a `find` command but had no results, or if the address book was empty).

This behaviour should be tested in a separate test case.

Reply:
Added tests for these cases.